themes/mcf/components/AboutClimateProjectExplorer.tsx

```diff
@@ -13,8 +13,8 @@ const AboutClimateProjectExplorer = () => {
           className="max-h-[824px] max-w-[1024px] object-cover"
         />{" "}
       </div>
-      <div className="relative lg:h-[824px] w-auto pb-16 lg:pb-0">
-        <div className="max-w-xl relative lg:z-10 ">
+      <div className="lg:relative lg:h-[824px] w-auto pb-16 lg:pb-0">
+        <div className="max-w-xl lg:relative lg:z-10 ">
```

Comment:
What is the `lg:` prefix?

Reply:
its the screen breakpoints i.e lg | 1024px

Reply:
fyi this tailwind css, not anything we have predefined in stylesheets or scss files.